codejail/jail_code.py

```diff
@@ -292,17 +292,14 @@ def jail_code(command, code=None, files=None, extra_files=None, argv=None,
 
         # Determine effective resource limits.
         effective_limits = get_effective_limits(limit_overrides_context)
-        limit_overrides_log_string = (
-            " (using overrides context '{}'')".format(limit_overrides_context)
-            if limit_overrides_context
-            else ""
-        )
-        log.info(
-            "Preparing to execute jailed code %s%s with resource limits: %r",
-            jail_code,
-            limit_overrides_log_string,
-            effective_limits,
-        )
+        if slug:
+            log.info(
+                "Preparing to execute jailed code %r "
+                "(overrides context = %r, resource limits = %r).",
+                slug,
+                limit_overrides_context,
+                effective_limits,
```

Comment:
The changes here:
* Log the `slug` (which is meant for logging) instead of `jail_code` (which is a function). This was just a silly mistake.
* Include the `limit_overrides_context` even if is `None`.
* Use `%r`, which unlike `%s`, wraps strings in single quotes for easier reading of the log message.